docs/trustee-tpm-setup.md

```diff
@@ -0,0 +1,205 @@
+# Introduction
+
+Instructions to use TPM attester for remote attestation with Trustee. This is primarily meant to
+experiment with Trustee attestation and resource policies based on TPM PCRs
+which can then be adapted for a confidential environment (eg. Azure CVM with vTPM).
+
+For creating fcos VM, please refer to the following [guide](../README.md).
+
+## Trustee server setup
+
+### Setup services
+
+Ensure you have `docker`, `docker-compose`, `rust`, `make` installed.
+
+Clone the repo.
+
+```sh
+git clone https://github.com/bpradipt/trustee.git
+cd trustee
+git checkout -b tpm-verifier origin/tpm-verifier
+```
+
+Setup authentication keys.
+We'll need these keys when using `kbs-client` to interact with Trustee services.
+
+```sh
+openssl genpkey -algorithm ed25519 > kbs/config/private.key
+openssl pkey -in kbs/config/private.key -pubout -out kbs/config/public.pub
+```
+
+Run Trustee services.
+
+This will start KBS, AS and RVPS services
+
+```sh
+docker-compose up -d
+```
+
+### Build and install kbs-client
+
+```sh
+cd kbs
+make cli
+sudo make install-cli
+```
+
+The `kbs-client` binary will be available at `/usr/local/bin/kbs-client`
+
+### Set KBS server address
+
+Get the IP address of the Trustee host.
+
+```sh
+device=$(ip r | awk '/^default/{print $5}')
+ip_address=$(test -n "$device" && ip addr show "$device" | awk '/inet /{print $2}' | cut -d/ -f1)
+export KBS_SERVER="$ip_address"
+echo "$KBS_SERVER"
```

Comment:
Super nit: this is valid only for the case when we have the kbc in the VM and trustee is outside, and the VM is using user networking, maybe it is worth mentioning it.